### DISCUSSION TITLE
Styling a FormControl with theme file at `baseStyle` level

### DISCUSSION THREAD

**Discussion by RyanPayso13** [chakra-ui/chakra-ui#4466] [Q&A] [answered]
Hi,

Have been using Chakra for a while but I can not get my head around styling all my components, in this case, `FormControl` at a global level with the theme file.

For example if i want to add margin bottom to my `FormControl` and `FormLabel` elements I would add the `components` object to the theme file like so:

```
  components: {
    Form: {
      parts: ['control', 'label', 'errorMessage'],
      baseStyle: {
        control: {
          marginBottom: '2rem',
        },
        label: {
          marginBottom: '3rem',
        },
      },
    },
  }, 
```

But this has no effect on the base style of the rendered `FormControl` or `FormLabel`.

Could someone please help me with what I am doing wrong here?

**Comment by RyanPayso13** [comment, marked as answer]
Having looked through the source code a bit more there is no `parts` array to `FormControl` as it is a `Context` rather than a component.  Therefore, it cannot be styled!

**Comment by aaronkik** [comment]
Hi Ryan,

It is possible to style the Form components. I've created an ugly looking form in this [sandbox](https://codesandbox.io/s/form-styling-2r6y9) to demonstrate. 

You can get an idea of how to style other components here: https://github.com/chakra-ui/chakra-ui/tree/main/packages/theme/src/components

You can also view the individual components on the chakra website and look for the **View theme source** button at the top of the page.
